src/game/objects/ai/trex.c

```diff
@@ -1,5 +1,7 @@
 #include "game/objects/ai/trex.h"
 
+#include "config.h"
```

Comment:
Again, kind of feels like over kill to pull the entirety of the config code in just for one check...

Reply:
It's OK